### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Mohsinsiddi**
PR for merging develop into main

### CODE REVIEW

**Comment by Mohsinsiddi** [comment]
Good to go for merging into develop

**Review by Mohsinsiddi — COMMENTED** [review]
Everything seems good

File: .env
Comment (nblogist) [inline .env:1]: pls remove api key

File: public/logo192.png
Comment (nblogist) [inline public/logo192.png:1]: let's remove these react icons

File: public/logo512.png
Comment (nblogist) [inline public/logo512.png:1]: let's remove these react icons

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import { useAccount, useNetwork } from "wagmi";
+
+export const ThemeContext = createContext(null);
+
+function App() {
+  const { chain } = useNetwork();
+  const { isConnected } = useAccount();
+  const [theme, setTheme] = useState("light");
+
+  // useEffect(() => {
```
Comment (nblogist) [inline src/App.js]: pls remove unused commented code

File: src/components/Activity/Activity.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            </tr>
+          </thead>
+          <tbody>
+            {Activities.map((c, i) => {
+              return (
+                <tr className={styles.clientrow}>
+                  <td>
+                    <img src={Profile} alt="Profile"></img>
+                  </td>
+                  <td>Hello</td>
```
Comment (nblogist) [inline src/components/Activity/Activity.js]: why are there hellos in here

File: src/components/Navbar/Navbar.js [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+  const [mobileOpen, setMobileOpen] = useState(false);
+
+  return (
+    <div className={theme === "light" ? styles.light : styles.dark}>
+      <div className={styles.entirenavbar}>
+        <div className={styles.left}>
+          <NavLink to="/">
+            <div className={styles.logo}>
+              <img src={logo} alt="Logo" />
+              {/* {theme === "light" ? (
```
Comment (nblogist) [inline src/components/Navbar/Navbar.js]: please remove unused commented code

File: src/components/Navbar/Navbar.js [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+                  Swap NFT
+                  <div></div>
+                </NavLink>
+              </li>
+            ) : (
+              ""
+            )}
+
+            {/* <li>
+                <NavLink
```
Comment (nblogist) [inline src/components/Navbar/Navbar.js]: removal of unused commented code

File: src/components/Popup/Popup.js [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+              network: Network.ETH_RINKEBY,
+            };
+            break;
+          case "maticmum":
+            config = {
+              apiKey: process.env.REACT_APP_ALCHEMY_API_KEY,
+              network: Network.MATIC_MUMBAI,
+            };
+            break;
+          case "Godwoken ":
```
Comment (nblogist) [inline src/components/Popup/Popup.js:61]: please make sure this extra space after Godwoken won't cause any bugs

File: src/components/Profile/CataloguePopup.js [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+          <div className={styles.info}>
+            <FcInfo />
+          </div>
+          <div className={styles.cross}>
+            <FaTimes
+              style={theme === "dark" ? { color: "black" } : { color: "white" }}
+              onClick={props.switch}
+            />
+            {/* {
+                            theme === 'dark' ? <img src={crossd} alt="" onClick={props.switch}></img> : <img src={cross} alt="" onClick={props.switch}></img>
```
Comment (nblogist) [inline src/components/Profile/CataloguePopup.js]: removal of unused commented code

File: src/components/Profiledesc/Profiledesc.js [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+              <a href={NFT_EXPLORER_URL} target="_blank">
+                <div className={styles.viewonexplorer}>
+                  View NFT on Explorer <BsBoxArrowUpRight />
+                </div>
+              </a>
+              <div className={styles.profdetails}>
+                <FaUserCircle className={styles.user} />
+                <div className={styles.profile}>
+                  <div className={styles.name}>{address}</div>
+                  {/* <div className={styles.userdesc}>From India</div> */}
```
Comment (nblogist) [inline src/components/Profiledesc/Profiledesc.js]: removal of unused commented code

File: src/components/Swap/SwapPopup.js [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+          <div className={styles.info}>
+            <FcInfo />
+          </div>
+          <div className={styles.cross}>
+            <FaTimes
+              style={theme === "dark" ? { color: "black" } : { color: "white" }}
+              onClick={props.switch}
+            />
+            {/* {
+                            theme === 'dark' ? <img src={crossd} alt="" onClick={props.switch}></img> : <img src={cross} alt="" onClick={props.switch}></img>
```
Comment (nblogist) [inline src/components/Swap/SwapPopup.js]: removal of unused commented code

File: src/components/SwapCatalogue/SwapCatalogue.js [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+    nft.selectedNFT_approve_count = 1;
+    nft.selectedNFT_swap_count = 1;
+    props.setSelected(nft);
+    props.setIndex(props.index + 1);
+  };
+  const { chain } = useNetwork();
+  let image_url = "";
+
+  try {
+    if (chain.network === "Godwoken ") {
```
Comment (nblogist) [inline src/components/SwapCatalogue/SwapCatalogue.js:25]: the space after Godwoken can cause unexpected bugs; please review

File: src/components/SwapCatalogue/SwapCatalogue.js [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+            <img
+              src={image_url}
+              alt="Sample"
+              className={styles.sampleimage}
+            ></img>
+          </div>
+          <div className={styles.lowerbox}>
+            <div className={styles.lowerboxcontent}>
+              <div className={styles.name}>{Truncate(props.nft.title)}</div>
+              {/* <div className={styles.desc}>By Someone</div> */}
```
Comment (nblogist) [inline src/components/SwapCatalogue/SwapCatalogue.js]: removal of unused commented code

File: src/components/SwapCatalogue/SwapCatalogue.module.scss [hunk 11]
```diff
@@ -0,0 +1,109 @@
...
+  margin-top: 8px;
+  border-radius: 20px;
+  padding: 5px 20px;
+  cursor: pointer;
+  background-color: transparent;
+  font-weight: 200;
+  font-family: "Roboto";
+}
+
+/* .bttn:hover{
```
Comment (nblogist) [inline src/components/SwapCatalogue/SwapCatalogue.module.scss]: removal of unused commented code

File: src/hooks/GetNFTs.js [hunk 12]
```diff
@@ -0,0 +1,34 @@
+// import { Alchemy, Network } from "alchemy-sdk";
```
Comment (nblogist) [inline src/hooks/GetNFTs.js]: removal of unused commented code/file

File: src/logo.svg
Comment (nblogist) [inline src/logo.svg:1]: let's remove this logo

File: src/utils/Toggle/Toggle.js [hunk 13]
```diff
@@ -0,0 +1,25 @@
...
+import { ThemeContext } from "../../App";
+
+function Toggle({ status, setStatus }) {
+    const inputref = useRef(status);
+    const themes = useContext(ThemeContext);
+    const { theme } = themes;
+
+    return (
+        <label className={styles.switch}>
+            {/* {status === "light" ?
```
Comment (nblogist) [inline src/utils/Toggle/Toggle.js]: removal of unused commented code

File: src/utils/Toggle/Toggle.module.css [hunk 14]
```diff
@@ -0,0 +1,139 @@
...
+        /* margin-right: 10px; */
+    }
+
+    .switchinputlight:checked~.switchhandle {
+        left: 23px;
+        background-image: -webkit-linear-gradient(top, #4d3bce 40%, #4d3bce);
+    }
+}
+
+/* @media only screen and (max-width:800px) {
```
Comment (nblogist) [inline src/utils/Toggle/Toggle.module.css]: removal of unused commented code

**Review by nblogist — CHANGES_REQUESTED** [review]
please review the comments. Good work! :rocket:

File: .env
Comment (Mohsinsiddi) [reply .env:1]: removed and added env.example file

File: public/logo192.png
Comment (Mohsinsiddi) [reply public/logo192.png:1]: removed

File: public/logo512.png
Comment (Mohsinsiddi) [reply public/logo512.png:1]: removed

File: src/App.js (hunk 1 above)
Comment (Mohsinsiddi) [reply src/App.js]: removed

File: src/components/Activity/Activity.js (hunk 2 above)
Comment (Mohsinsiddi) [reply src/components/Activity/Activity.js]: Updated it

File: src/components/Navbar/Navbar.js (hunk 3 above)
Comment (Mohsinsiddi) [reply src/components/Navbar/Navbar.js]: removed them

File: src/components/Navbar/Navbar.js (hunk 4 above)
Comment (Mohsinsiddi) [reply src/components/Navbar/Navbar.js]: removed them

File: src/components/Popup/Popup.js (hunk 5 above)
Comment (Mohsinsiddi) [reply src/components/Popup/Popup.js:61]: yeah its working fine

File: src/components/Profile/CataloguePopup.js (hunk 6 above)
Comment (Mohsinsiddi) [reply src/components/Profile/CataloguePopup.js]: removed them

File: src/components/Profiledesc/Profiledesc.js (hunk 7 above)
Comment (Mohsinsiddi) [reply src/components/Profiledesc/Profiledesc.js]: removed

File: src/components/Swap/SwapPopup.js (hunk 8 above)
Comment (Mohsinsiddi) [reply src/components/Swap/SwapPopup.js]: removed

File: src/components/SwapCatalogue/SwapCatalogue.js (hunk 9 above)
Comment (Mohsinsiddi) [reply src/components/SwapCatalogue/SwapCatalogue.js:25]: Reviewed, its all good here!

File: src/components/SwapCatalogue/SwapCatalogue.js (hunk 10 above)
Comment (Mohsinsiddi) [reply src/components/SwapCatalogue/SwapCatalogue.js]: removed

File: src/components/SwapCatalogue/SwapCatalogue.module.scss (hunk 11 above)
Comment (Mohsinsiddi) [reply src/components/SwapCatalogue/SwapCatalogue.module.scss]: removed

File: src/hooks/GetNFTs.js (hunk 12 above)
Comment (Mohsinsiddi) [reply src/hooks/GetNFTs.js]: deleted the unused file

File: src/logo.svg
Comment (Mohsinsiddi) [reply src/logo.svg:1]: removed

File: src/utils/Toggle/Toggle.js (hunk 13 above)
Comment (Mohsinsiddi) [reply src/utils/Toggle/Toggle.js]: removed

File: src/utils/Toggle/Toggle.module.css (hunk 14 above)
Comment (Mohsinsiddi) [reply src/utils/Toggle/Toggle.module.css]: removed

**Comment by Mohsinsiddi** [comment]
Also removed all unused variables and imports

**Review by Mohsinsiddi — COMMENTED** [review]
Good to go now

**Review by nblogist — APPROVED** [review]
LGTM!

**Pull request merged**